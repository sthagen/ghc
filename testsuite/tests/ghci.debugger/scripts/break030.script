:set -XDeriveDataTypeable
:l break030
:set -fno-break-on-error
:set -fbreak-on-exception
exception_uncaught
:force _exception
:co
