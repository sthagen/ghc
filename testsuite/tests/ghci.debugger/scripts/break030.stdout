Stopped in <exception thrown>, <unknown>
_exception :: e = SomeExceptionWithLocation
                    (GHC.IO.Exception.IOError Nothing GHC.IO.Exception.UserError ....)
                    []
_exception = SomeExceptionWithLocation
               (GHC.IO.Exception.IOError
                  Nothing GHC.IO.Exception.UserError [] "error" Nothing Nothing)
               []
Stopped in <exception thrown>, <unknown>
_exception :: e = SomeExceptionWithLocation
                    (GHC.IO.Exception.IOError Nothing GHC.IO.Exception.UserError ....)
                    []
